multiversum.cls

```diff
@@ -65,141 +87,111 @@
 %
 % Replace math greek fonts by Source Serif Pro (No Italics available -> Use fake italics)
 \setmathfont(Greek)[
-UprightFont={* Light},
-BoldFont={* Semibold},
-AutoFakeSlant=0.15
+  UprightFont={* Light},
+  ItalicFont={* Light Italic},
+  BoldFont={* Semibold},
+  BoldItalicFont={* Semibold Italic},
+  Scale=MatchLowercase
 ]
 {Source Serif Pro}
 %
 %%%
 
-%%% Colors!
-\usepackage[table]{xcolor}
-%%%
 
-%%%
-% Memoir Class Page formatting
-%
-% Compute a suitable line width.
-\setlxvchars \setxlvchars
-% type block size:
-\settypeblocksize{700pt}{1.6\lxvchars}{*}
-% type block at the center of the page:
-\setlrmargins{*}{*}{1}
-\setulmargins{*}{*}{1}
-% separator between two columns, no rule:
-\setcolsepandrule{0.5cm}{0pt}
-% header and footer height:
-\setheadfoot{2\onelineskip}{2\onelineskip}
-% note margin width and separator:
-\setmarginnotes{17pt}{3.5cm}{\onelineskip}
-% save changes and fix the layout:
-\checkandfixthelayout
-% empty page layout (no header, no footer)
-\pagestyle{empty}
-%
+%%% Colors!
+\PassOptionsToPackage{table}{xcolor}
+\RequirePackage{xcolor}
 %%%
 
 
 %%%
-% Balance last two columns
-% Set "balance" variable to "true" in pandoc to balance the columns on the last
-% page.
-%$if(balance)$\usepackage{flushend}$endif$
+% Divisions
+% Section Style
+\setbeforesecskip{-.5em}
+\newcommand{\ruledsec}[1]{%
+	\LARGE\scshape\centering #1 \rule{\linewidth}{0.4pt}\noindent}
+\setsecheadstyle{\ruledsec}
+\setaftersecskip{.5em}
+% Subsection Style
+\setsubsecheadstyle{\Large\normalfont\raggedright}
+% Paragraph Style
+\setparaheadstyle{\bfseries}
+% Disable numbering
+\setsecnumdepth{part}
+% include subsections in PDF bookmarks
+\setcounter{tocdepth}{2}
+%
 %%%
 
 
 %%%
 % Language
-\usepackage[english,ngerman]{babel}
-\usepackage[german=quotes]{csquotes}
+\PassOptionsToPackage{english,ngerman}{babel}
+\RequirePackage{babel}
+\PassOptionsToPackage{german=quotes}{csquotes}
+\RequirePackage{csquotes}
 %
 %%%
 
 
 %%%
 % Figures
-\usepackage{graphicx}
+\RequirePackage{graphicx}
 %
 %%%
 
+
 %%%
 % Date Format:
-\usepackage[ngerman]{datetime}
+\PassOptionsToPackage{ngerman}{datetime}
+\RequirePackage[ngerman]{datetime}
```

Comment:
```suggestion
\RequirePackage{datetime}
```
I am reasonably sure it would work if you do both ```\PassOptionsToPackage{ngerman}{datetime}``` and ```\RequirePackage[ngerman]...```, but wouldn't rely on it.

Reply:
I don't think the way it is hurts and it is probably more stable. Thank you for the suggestion though.